From 1704b021b3d662b6f2e64208370c5a944144531d Mon Sep 17 00:00:00 2001
From: techtino <w1682146@my.westminster.ac.uk>
Date: Sat, 30 Apr 2022 14:06:55 +0100
Subject: [PATCH] Fix orientation detection for Asus Transformer T100TAF,
 copied T100TA rule

(cherry picked from commit ad44259225eabeb9755440506c261c6080a2f405)

Related: #2087778
---
 hwdb.d/60-sensor.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index d6a584150c..0dfaaaf467 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -123,6 +123,7 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:pnT300CHI:*
 
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnM80TA:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnT100TA:*
+sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnT100TAF:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:pnT200TA:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
